generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.ts.ejs

```diff
@@ -28,28 +30,32 @@ import { PasswordResetFinishService } from './password-reset-finish.service';
     templateUrl: './password-reset-finish.component.html'
 })
 export class PasswordResetFinishComponent implements OnInit, AfterViewInit {
-    confirmPassword: string;
     doNotMatch: string;
     error: string;
     keyMissing: boolean;
-    resetAccount: any;
     success: string;
     modalRef: NgbModalRef;
     key: string;
 
+    passwordForm = this.fb.group({
+        newPassword: ['', [Validators.required, Validators.minLength(4), Validators.maxLength(50)]],
+        confirmPassword: ['', [Validators.required, Validators.minLength(4), Validators.maxLength(50)]],
```

Comment:
Does this works? I always used `Validators.compose` to use multiple validators...

Reply:
Yes, it works, do you prefer compose?

Reply:
No this way looks clearer for me so that's ok 😄